server/routers/share-recovery/recoverWallet.ts

```diff
@@ -99,27 +106,63 @@ export const recoverWallet = protectedProcedure
       });
     }
 
-    const publicKey =
-      recoveryKeyShare?.recoveryBackupSharePublicKey ||
-      (
-        await ctx.prisma.wallet.findFirst({
-          select: { publicKey: true },
-          where: {
-            id: input.walletId,
-            userId: ctx.user.id,
-          },
-        })
-      )?.publicKey ||
-      null;
-
-    const isChallengeValid = await ChallengeUtils.verifyChallenge({
-      challenge,
-      session: ctx.session,
-      shareHash: recoveryKeyShare?.recoveryBackupShareHash || null,
-      now,
-      solution: input.challengeSolution,
-      publicKey,
-    });
+    // Check if this is a cross-auth recovery attempt
+    const isCrossAuthRecovery = input.crossAuthRecovery === true;
+    
+    // Log auth provider and the cross-auth recovery flag
+    if (isCrossAuthRecovery) {
```

Comment:
What is cross auth recovery?

Reply:
left behind logic when i was testing passkey with the Wander UI and having the backup file "shareable" but as per discussion, that is not something we would want to support now.